spec/rubocop/cop/layout/end_alignment_spec.rb

```diff
@@ -86,13 +176,35 @@
       { 'EnforcedStyleAlignWith' => 'variable', 'AutoCorrect' => true }
     end
 
-    include_examples 'misaligned', '', 'class',  'Test',      '  end'
-    include_examples 'misaligned', '', 'module', 'Test',      '  end'
-    include_examples 'misaligned', '', 'if',     'test',      '  end'
-    include_examples 'misaligned', '', 'unless', 'test',      '  end'
-    include_examples 'misaligned', '', 'while',  'test',      '  end'
-    include_examples 'misaligned', '', 'until',  'test',      '  end'
-    include_examples 'misaligned', '', 'case',   'a when b',  '  end'
+    include_examples 'misaligned', <<-RUBY, false
+      class Test
+        end
+        ^^^ `end` at 2, 2 is not aligned with `class` at 1, 0.
+
+      module Test
+             end
+             ^^^ `end` at 2, 7 is not aligned with `module` at 1, 0.
+
+      if test
+        end
+        ^^^ `end` at 2, 2 is not aligned with `if` at 1, 0.
+
+      unless test
+        end
+        ^^^ `end` at 2, 2 is not aligned with `unless` at 1, 0.
+
+      while test
+        end
+        ^^^ `end` at 2, 2 is not aligned with `while` at 1, 0.
+
+      until test
+        end
+        ^^^ `end` at 2, 2 is not aligned with `until` at 1, 0.
+
+      case a when b
+        end
+        ^^^ `end` at 2, 2 is not aligned with `case` at 1, 0.
```

Comment:
Possibly add one 'variable' example that doesn't need to be re-aligned to a 0 column.

Reply:
Whoops! Nevermind :).